src/main/java/ch/njol/skript/expressions/ExprPlayerChatCompletions.java

```diff
@@ -0,0 +1,113 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Player Chat Completions")
+@Description({
+	"The custom chat completion suggestions. You can add, set, remove, and delete them. Removing online players name with this expression is ineffective.",
+	"This expression will not return anything due to Bukkit limitations."
+})
+@Examples({
+	"add \"Skript\" and \"Njol\" to chat completions of all players",
+	"remove \"text\" from {_p}'s chat completions",
+	"delete player's chat completions"
+})
+@RequiredPlugins("Spigot 1.19+")
```

Comment:
Remain unchanged due to clarification from skriptlang channel.
![image](https://github.com/SkriptLang/Skript/assets/72163224/660eafc8-1786-4453-b9c5-e34ef982243a)

Reply:
Well, we use MC as a reference to Minecraft but since we don't currently have a specific convention about this we can skip it for now as it's not a big deal either. Later on we can change it all together in another PR.